Scripts/Runtime/Util/Random/RandomUtil.cs

```diff
@@ -0,0 +1,45 @@
+using System.Runtime.CompilerServices;
+using Unity.Mathematics;
+using DateTime = System.DateTime;
+using RuntimeInitializeLoadType = UnityEngine.RuntimeInitializeLoadType;
+
+namespace Anvil.Unity.DOTS.Util
+{
+    /// <summary>
+    /// Utility class for simplifying getting access to a <see cref="Random"/> instance.
+    /// </summary>
+    public static class RandomUtil
+    {
+        private static Random s_InternalRandom;
+
+        //With DOTS, it's recommended to disable the Domain Reload when entering play mode as it will be pretty slow.
+        //https://docs.unity3d.com/Packages/com.unity.entities@0.17/manual/install_setup.html
+        //Because of this, statics will preserve state across play sessions. By using the RuntimeInitializeOnLoadMethod
+        //we can ensure we reset the state on every play session.
+        //RuntimeInitializeLoadType.SubsystemRegistration represents the earliest moment of all the entries in the enum.
```

Comment:
```suggestion
        // With the Burst compiler, it's common to disable the Domain Reload step when entering play mode to increase iteration time.
        // https://docs.unity3d.com/Packages/com.unity.entities@0.17/manual/install_setup.html
        // Because of this, statics will preserve state across play sessions.
        // `RuntimeInitializeOnLoadMethod` ensures we reset the state on every play session.
        // `RuntimeInitializeLoadType.SubsystemRegistration` represents the earliest moment of all the entries in the enum.
```

Reply:
Cleaned this up but I don't know if we need to document this as thoroughly all the time. A quick
```
// Facilitates state reset when DomainReload is disabled
// <Link>
```

Is probably enough. Not saying you delete this just something to prevent us writing DOTS tutorials on every implementation!

Reply:
Changed to your suggestion.

Yeah I was on the fence with how much I write in the code vs the PR vs lost forever to our slack history until we pay for it. 

I have found it helpful currently to over-document just to make sure our own understanding is correct and maybe I'm just over compensating for the very sparse documentation that currently exists in Unity's docs.